hasher-matcher-actioner/hmalib/lambdas/api/datasets_api.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
I really like that you made this its own file. This a good convention we should start to follow universally